### DISCUSSION TITLE
Rig creation example

### DISCUSSION THREAD

**Discussion by LeXXik** [jrouwe/JoltPhysics.js#118] [Q&A] [answered]
The rig creation example shows how to create a rig. It uses a few methods which are static. This makes me assume that the last one would overwrite the previous value? For example:

https://github.com/jrouwe/JoltPhysics.js/blob/6d900220ab32c52e9a11060c4e7ea959b2a6f2c2/Examples/rig/create_rig.html#L111-L112

Would the lower right arm overwrite lower left arm value here?

**Comment by jrouwe** [comment, marked as answer]
Yes, you're right. This is wrong. It works because sAxisX() has a static variable and Mul() has another variable and all the Mul's are Mul(-1) so they all have the same value.

**Comment by jrouwe** [comment]
Fixed in https://github.com/jrouwe/JoltPhysics.js/commit/49283254d2063ff9163085e0486dcc5f2f873753
